proposals/0400-init-accessors.md

```diff
@@ -491,13 +491,22 @@ init() {
 }
 ```
 
+### Restrictions
+
+* A property with an `init` accessor must be declared in the same source file 
+  as the respective type declaration.
+* An `init` accessor is always `fileprivate` and thus cannot be called from
+  within a different source file.
+
```

Comment:
I am still trying to make sense of the first one (it was inferred from [this](https://forums.swift.org/t/pitch-init-accessors/64881/16) post). If an `init` accessor is always confined to its file, then, logically, it cannot break a memberwise initializer generated in another file that isn’t aware of its existence.

Reply:
I think this should just say that init accessors can only be written in the primary declaration of a type.

Reply:
```suggestion
### Restrictions

A property with an `init` accessor can only be declared in the primary 
declaration of a type.

```